pkg/limayaml/defaults.go

```diff
@@ -551,9 +562,24 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 		y.MountType = o.MountType
 	}
 	if y.MountType == nil || *y.MountType == "" {
```

Comment:
Since we seem to support `"default"` for `vmType`, should we support it for `mountType` as well (would just be an alias to the empty string).

Reply:
Mount type was a string before, so old templates/instances used "" (not null)

Reply:
added "default"